.github/workflows/build.yml

```diff
@@ -1,41 +1,47 @@
-name: Build
-on: [push, pull_request]
+name: Build Code
+
+on:
+  push:
+  pull_request:
+    types: [opened]
+
+env:
+  INEXOR_VULKAN_VERSION: "1.3.216.0"
+  INEXOR_VULKAN_SDK_PATH: "$GITHUB_WORKSPACE/../vulkan_sdk/"
 
 jobs:
   linux:
     name: ${{ matrix.config.name }}
     runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'pull_request' && github.event.action == 'opened' && !github.event.pull_request.merged || github.event_name == 'push' }}
     container: ubuntu:rolling
     env:
       DEBIAN_FRONTEND: "noninteractive"
-      CONAN_USER_HOME: "${{ github.workspace }}/conan/"
-      CONAN_USER_HOME_SHORT: "${{ github.workspace }}/conan/s"
-      inexor_conan_path: "$HOME/.local/bin"
     strategy:
       fail-fast: false
       matrix:
         config:
           - {
             name: "Ubuntu Clang",
-            cc: "clang", cxx: "clang++",
-            cmake_configure_options: '-DCMAKE_EXE_LINKER_FLAGS="-v -fuse-ld=lld"',
-            build_type: "Release",
+            compiler: "clang",
+            cc: "clang-15", cxx: "clang++-15",
           }
           - {
             name: "Ubuntu GCC",
-            cc: "gcc", cxx: "g++",
-            build_type: "Release",
+            compiler: "gcc",
+            cc: "gcc-12", cxx: "g++-12",
           }
 
     steps:
       - name: Update environment
         shell: bash
         run: |
           # Update package lists
-          apt update -qq
+          apt-get update -qq
```

Comment:
`apt` should work fine?

Reply:
I added it, let's try

Reply:
You were right, I can use only `apt update -qq`.